### DISCUSSION TITLE
Commonsense Knowledge, Symbolic Knowledge Distillation paper: Extrinsic Evaluations

### DISCUSSION THREAD

**Discussion by mayanks43** [eduardoblanco/csc696H-Spring24#14] [Q&A]
What is the point of ATOMIC expansion? Where does it help? What extrinsic evaluations could have been done?

**Comment by eduardoblanco** [comment]
If you don't "volunteer" to participate you will end up tagged with the harder question that nobody has answered yet...

@bhavya2109sharma Ideas?

**Comment by bhavya2109sharma** [comment]
The process of enriching commonsense knowledge graphs using the ATOMIC dataset is known as the "ATOMIC expansion." According to me expanding commonsense knowledge helps in more concise and better results.
I can think of these 2 extrinsic evaluations: Performance in downstream tasks where it can be tested and the Comparative Analysis with the  model which is trained with ATOMIC expansion and the one with no ATOMIC expansion
